cou/cli.py

```diff
@@ -145,6 +151,15 @@ async def run_upgrade(
     analysis_result, upgrade_plan = await analyze_and_plan(model_name, backup_database)
     logger.info(upgrade_plan)
 
+    # NOTE(rgildein): add handling upgrade plan canceling for SIGINT (ctrl+c) and SIGTERM
+    loop = asyncio.get_event_loop()
+    loop.add_signal_handler(
+        SIGINT, keyboard_interrupt_handler, upgrade_plan, loop, progress_indicator
+    )
+    loop.add_signal_handler(
+        SIGTERM, keyboard_interrupt_handler, upgrade_plan, loop, progress_indicator
```

Comment:
If I understood correctly SIGINT is when the user hits ctrl+c. When `SIGTERM` is going to happen?

Reply:
For example when I tried kill process in htop, the default value is `SIGTERM` (you can use different one also, like `SYSKILL`). If I'm correct the `SIGTERM` happen also when you close your terminal.